graylog2-server/src/main/java/org/graylog2/alerts/StaticEmailAlertSender.java

```diff
@@ -69,10 +79,10 @@ public void sendEmails(Stream stream, AlertCondition.CheckResult checkResult) th
     private void sendEmail(String emailAddress, Stream stream, AlertCondition.CheckResult checkResult, List<Message> backlog) throws TransportConfigurationException, EmailException {
         LOG.debug("Sending mail to " + emailAddress);
         if(!configuration.isEnabled()) {
-            throw new TransportConfigurationException("Email transport is not enabled!");
+            throw new TransportConfigurationException("Email transport is not enabled in configuration file!");
```

Comment:
I would change the exclamation mark to a full-stop. Sounds less aggressive/panicky.